src/image.rs

```diff
@@ -41,6 +41,7 @@ use crate::utils::{
     types::Index,
 };
 
+#[cfg_attr(nightly, allow(clippy::upper_case_acronyms))]
```

Comment:
Thank you for contribution!
I just have a question about this clippy lint, which is used for nightly toolchain.
If we fix our CI to stable, as @denisxor mentioned, do we still need `rustc_version` and this one rule?

Reply:
If your objective is to fix CI, then you're correct, you don't need to populate `rustc_version` or use this rule.

If your objective is to be able to pick up this code base and run a nightly version of `clippy` locally, you still do. Note that this could be useful, because [the nightly version allows developers to fix clippy lints automatically](https://github.com/rust-lang/rust-clippy#automatically-applying-clippy-suggestions).

Reply:
Didn't know about this one feature, can be useful for later changes.